conf/pf.conf.defaults

```diff
@@ -757,6 +757,31 @@ pfcmd_warning_color=yellow
 #
 # Color of the success text for pfcmd
 pfcmd_success_color=green
+#
+# advanced.use_omapi_to_lookup_mac
+#
+# Use omapi to query dhcpd for the mac address of an ip
+use_omapi_to_lookup_mac=disabled
```

Comment:
We should be then document when to disable it.
For example a user use their own dhcp server that does not have omapi support or not configured.